client/app/telecom/pack/xdsl/access/ipv6/pack-xdsl-access-ipv6.controller.js

```diff
@@ -34,8 +34,10 @@ angular.module('managerApp').controller('XdslAccessIpv6Ctrl', function ($statePa
       $scope.access.xdsl.ipv6Enabled = !$scope.access.xdsl.ipv6Enabled;
     }
 
-    $scope.message = "<h3>{{ 'xdsl_access_ipv6_status' | translate }}</h3>{{ access.xdsl.ipv6Enabled ? 'xdsl_access_ipv6_really_on' : 'xdsl_access_ipv6_really_off' | translate }} "
-    + "<div class='text-warning'>{{ 'xdsl_access_ipv6_warning' | translate }}</div> ";
+    const title = $translate.instant($scope.access.xdsl.ipv6Enabled ? 'xdsl_access_ipv6_status' : 'xdsl_access_ipv6_status_off');
+    const question = $translate.instant($scope.access.xdsl.ipv6Enabled ? 'xdsl_access_ipv6_really_off' : 'xdsl_access_ipv6_really_on');
+    $scope.message = `<h3>${title}</h3> ${question}
```

Comment:
Do you think you could extract the HTML in an HTML file instead ? :)

Reply:
To have correct messages, yes :p